service/history/handler/interface.go

```diff
@@ -86,4 +86,5 @@ type Handler interface {
 	SyncShardStatus(context.Context, *types.SyncShardStatusRequest) error
 	TerminateWorkflowExecution(context.Context, *types.HistoryTerminateWorkflowExecutionRequest) error
 	GetFailoverInfo(context.Context, *types.GetFailoverInfoRequest) (*types.GetFailoverInfoResponse, error)
+	RatelimitUpdate(ctx context.Context, Request *types.RatelimitUpdateRequest) (*types.RatelimitUpdateResponse, error)
```

Comment:
do we have a way to avoid non-cadence services calling this?

Reply:
No more than any other history-API method